doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -41,22 +41,22 @@ Synopsis
 Options
 ~~~~~~~
 
-  -f, --format <html|xml> Specifies whether to generate an HTML or XML
-                          output file. The default output `format` and the 
-                          default output `specdocformat` is got from the
-                          output file extension.
-                          :file:`*.html`    -> `-f HTML -s HTML`
-                          :file:`*.xml`     -> `-f XML  -s RAW`
-                          :file:`*.libspec` -> `-f XML  -s HTML`
-   -s --specdocformat <raw|html>
+  -f, --format <html|xml>
+                          Specifies whether to generate an HTML, XML or
+                          JSON output file. The default output `format`
+                          and the default output `specdocformat` is fetched
+                          from the output file extension.
+                          :file:`*.html` -> `-f HTML -s HTML`,
+                          :file:`*.xml` -> `-f XML -s RAW`,
+                          :file:`*.json` -> `-f JSON -s HTML` and
+                          :file:`*.libspec` -> `-f XML -s HTML`
+  -s, --specdocformat <raw|html>
                           Specifies for XML outputs whether the keyword
                           documentation is converted to HTML regardless
                           of the original documentation format or kept
-                          raw as in the library or source spec file.
-                          :file:`*.html` files always get HTML documentation
-                          format. Default is based on the format.
-                          `--format HTML` -> `HTML`
-                          `--format XML`  -> `RAW`
+                          raw as in the library, resource or source spec file.
+                          :file:`*.html` files only accept `HTML`.
+                          Default is based on the format.
```

Comment:
Default values should be explained here. I think the whole doc could be something like this:

---

Specifies the documentation format to use with XML and JSON spec files. `HTML` means converting the documentation to HTML regardless the original documentation format and `RAW` means keeping the original format. The default is got based on the `--format` option so that with XML the default is `RAW` and with JSON it is `HTML`. If `--format` is not used, the final default is got based on the output file extension so that with `*.xml` default is `RAW` and with `*.libspec` and `*.json` it is `HTML`.

---

Reply:
can you please fix the docs so that they mach your taste, please?

i think

> Default is based on the `format`.

give the user the hint to just look one line to the top and would see the defaults.